pkg/lang/ir/python.go

```diff
@@ -54,14 +54,16 @@ func (g Graph) compilePython(aptStage llb.State) (llb.State, error) {
 	}
 	builtinSystemStage := pypiMirrorStage
 
+	systemStage := g.compileSystemPackages(builtinSystemStage)
+
 	sshStage, err := g.copySSHKey(builtinSystemStage)
```

Comment:
Could we move systemStage after ssh and shell configuration?

Or the ssh cache will not work after the apt packages are changed.